### DISCUSSION TITLE
Newer UI?

### DISCUSSION THREAD

**Discussion by dreemur99** [LostRuins/koboldcpp#402] [Ideas]
Any chance KoboldCPP will get the newer UI (like 0cc4m's version) so we can have more control over outputs?

**Comment by LostRuins** [comment]
The UI for kobold united can be used for kobold cpp. Just launch that, and connect it as an external KAI endpoint.

**Reply by dreemur99** [reply]
Only partially works. Sometimes gives errors and streaming doesnt work at all.

**Reply by LostRuins** [reply]
What do you like about the United UI that Lite does not have?

**Reply by dreemur99** [reply]
/most liked/
Seeing all the possible/past generations on the side and being able to switch between them (when you hit back they are listed on the side).
The expanded world info keyword things. Being able to list multiple keywords per entry.
Direct control over randomness/seed.
User scripts. still learning how to make and use them but its a cool feature to have. i thought koboldcpp had it but i must have been mistaken
/less important/
Direct control over biasing
The options under the Settings>Other
The general look an feel of it.
